jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -194,15 +194,15 @@ def test_numpy_cube(cubeviz_helper):
     assert data.label == 'Array'
     assert data.shape == (4, 3, 2)  # x, y, z
     assert isinstance(data.coords, PaddedSpectrumWCS)
-    assert flux.units == 'ct'
+    assert flux.units == 'ct / pix2'
```

Comment:
I think this fixes a bug where pure numpy cube does not get that PIX2 assigned like all the other stuff loaded into Cubeviz, right?

Reply:
Nice catch! I just checked this on main and it wasn't being coerced